### PR TITLE
feat(lowercase-name): add 'ignore' option (#76)

### PR DESCRIPTION
**Pull request by lionize**
This PR adds the `ignore` option discussed in #76.

### CODE REVIEW

File: rules/lowercase-name.js [hunk 1]
```diff
@@ -54,11 +54,21 @@ module.exports = {
     },
   },
   create(context) {
+    const ignoredFunctionNames = (
+      context.options[0] || { ignore: [] }
```
Comment (SimenB) [inline rules/lowercase-name.js]: what happens here if `context.options[0] = {}`?

Comment (lionize) [reply rules/lowercase-name.js]: @SimenB fixed

**Comment by lionize** [comment]
Also added a check for empty argument list which could throw an error.

**Comment by mchandler-cng** [comment]
@SimenB fixed

**Comment by mchandler-cng** [comment]
@SimenB 👍

**Pull request merged**